packages/twenty-front/src/modules/activities/emails/components/EmailThreadMessage.tsx

```diff
@@ -60,7 +60,7 @@ export const EmailThreadMessage = ({
       </StyledThreadMessageHeader>
       <StyledThreadMessageBody>
         {isOpen ? (
-          <EmailThreadMessageBody body={body} />
+          <EmailThreadMessageBody body={body} isDisplayed={isOpen} />
```

Comment:
This one will always be true or am I missing something? @thomtrp

Reply:
Good point @Weiko! updated 👍